Packs/CadoResponse/Integrations/CadoResponse/README.md

```diff
@@ -0,0 +1,624 @@
+Automate data collection. Process data at cloud speed. Analyze with purpose.
+This integration was integrated and tested with version 1.1.0 of CadoResponse
+
+## Configure Cado Response on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Cado Response.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Cado Response Instance | The URL for your Cado Response Instance | True |
+    | API Key | The API Key to use for connection | True |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+    | Default Cado Response Project | Provides a fallback Project ID from Cado Response if you forget to add an ID to a command. If you're not sure, don't change\! | True |
+    | Default AWS Region | Provides a default AWS region to fallback to if you forget to add it to a command. | True |
+    | Default S3 Bucket | Provides a default S3 bucket to fallback to if you forget to add it to a command. | True |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### cado-create-project
+***
+Create a project in Cado Response
+
+
+#### Base Command
+
+`cado-create-project`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| project_name | Name of the project. | Required | 
+| project_description | Description for the project. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| CadoResponse.Project.id | Number | The Project ID of the newly created project | 
+
+
+#### Command Example
+``` ```
```

Comment:
why is there no example with results? can you please add it manually or use demisto-sdk generate-docs command to do so